google_compute_engine/accounts/oslogin_utils.py

```diff
@@ -103,37 +103,50 @@ def _RemoveOsLoginNssCache(self):
         if e.errno != errno.ENOENT:
           raise
 
-  def UpdateOsLogin(self, enable_oslogin, two_factor=False):
+  def UpdateOsLogin(self, oslogin_desired, two_factor_desired=False):
     """Update whether OS Login is enabled and update NSS cache if necessary.
 
     Args:
-      enable_oslogin: bool, enable OS Login if True, disable if False.
+      oslogin_desired: bool, enable OS Login if True, disable if False.
```

Comment:
two_factor_desired is an undocumented arg

Reply:
Thanks!